Procfile

```diff
@@ -0,0 +1 @@
+web: bundle exec rake cf:on_first_instance db:migrate && bundle exec puma
```

Comment:
It looks for `config/puma.rb` by default:

https://github.com/puma/puma#configuration-file